### DISCUSSION TITLE
Windows Installation with Forbidding PowerShell ExecutionPolicy

### DISCUSSION THREAD

**Discussion by DKGH** [microsoft/vcpkg#35291] [Q&A] [answered]
My company forbids running unsigned powershell scripts. How do I install vcpkg from source in this case. I don't want to use Visual Studio vcpkg.

**Comment by dg0yt** [comment]
The powershell script doesn't do much magic.
Anyways, did you try to `Set-ExecutionPolicy -Scope Process -ExecutionPolicy RemoteSigned` first? (From my memory, can't check now.)

**Reply by DKGH** [reply]
Yes, I tried setting policy at different scopes. I also tried gpedit. I can't seem to alter machine policy. I also tried everything in admin mode. Is there a way to compile and build vcpkg itself? Can't find an appropriate CMakeLists.txt for that.

**Reply by dg0yt** [reply, marked as answer]
There is https://github.com/microsoft/vcpkg-tool.

**Reply by DKGH** [reply]
What's the relationship between vcpkg and vcpkg-tools? Do I have to explicitly copy ports from vcpkg to my manually built vcpkg-tools?

**Reply by dg0yt** [reply]
No, you have to copy the tool to the root of the vcpkg git checkout. (The bootstrap script would download a tool binary. You could also download manually.)

**Reply by Osyotr** [reply]
@BillyONeal on discord said that there was a reason for bootstrap script to be unsigned, but not anymore. I suggest you convert this to a proper issue.
